src/Features/CustomerEffortScoreTracks.php

```diff
@@ -234,4 +269,27 @@ public function run_on_update_options() {
 			)
 		);
 	}
+
+	/**
+	 * Determine on initiating CES survey on searching for product or orders.
+	 */
+	public function run_on_load_edit_php() {
+		$allowed_types = array( 'product', 'shop_order' );
+
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		// We're only interested for certain post types.
+		if ( empty( $_GET['post_type'] ) || ! in_array( $_GET['post_type'], $allowed_types, true ) ) {
```

Comment:
Could [`get_current_screen()`](https://developer.wordpress.org/reference/functions/get_current_screen/) be used here to find the current post type?

Reply:
Made the change in d1fb96b. Thanks for the suggestion @adrianduffell!